frontend/src/patientApp/PxpApiService.ts

```diff
@@ -54,4 +55,18 @@ export class PxpApi {
       }),
     }).then((res) => res.json());
   }
+
+  static sendSmsLink(patientLinkId: string) {
+    return fetch(`${API_URL}/send-patient-link`, {
+      method: "post",
+      mode: "cors",
+      headers: {
+        "Content-Type": "application/json",
+        Accept: "application/json",
+        "Access-Control-Request-Headers": "Authorization",
+        Authorization: `Bearer ${localStorage.getItem("access_token")}`
+      },
+      body: String(patientLinkId),
```

Comment:
Nice - I like how this only sends the plid to the backend, not the phone number, to avoid security concerns w/ roles etc

Reply:
🙇